packages/governance/src/param-manager.js

```diff
@@ -0,0 +1,106 @@
+// @ts-check
+
+import { makeStore } from '@agoric/store';
+import { assert, details as X } from '@agoric/assert';
+import { Nat } from '@agoric/nat';
+import { assertIsRatio } from '@agoric/zoe/src/contractSupport';
+import { AmountMath, looksLikeBrand } from '@agoric/ertp';
+
+const ParamType = {
+  AMOUNT: 'amount',
+  ANY: 'any',
+  BIGINT: 'bigint',
+  BRAND: 'brand',
+  HANDLE: 'handle',
+  NAT: 'nat',
+  RATIO: 'ratio',
+  STRING: 'string',
+};
+harden(ParamType);
+
+const assertType = (type, value, name) => {
+  if (!type) {
+    // undefined type means don't verify. Did we omit an interesting type?
+    return;
+  }
```

Comment:
There's already an escape hatch though: `ParamType.ANY`. Is there a reason why the 3rd parties can't just use that? Allowing the type to be undefined seems like it will allow for user error.